src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.3.0")]
+[assembly: AssemblyFileVersion("1.0.4.0")]
```

Comment:
Since this is an additive change, please bump the minor version. Please make sure this is reflected elsewhere too

Reply:
resolved, bump minor version, now version 1.1.3